starfish/models/squid_model.py

```diff
@@ -78,6 +80,21 @@ def register_asset(self, metadata, account):
         squid_ocean = self.get_squid_ocean(account)
         return squid_ocean.assets.create(metadata, account)
 
+    def validate_metadata(self, metadata):
+        """
+
+        Validate the metadata with plesto
+
+        :param dict metadata: metadata to validate
+        :return: True if the metadata is valid
+        :type: boolean
+
+        """
+        # fix codacy to stop making this a static function
+        if self._ocean:
+            return is_valid_dict_local(metadata)
```

Comment:
I foresee an issue here: the invoke operation metadata may not be compliant with plecos.is_valid_dict_local. I presume that in the near term future, invokable operation's metadata can only be registered on Surfer?

Reply:
At the moment I'm not forcing the validation, but we may have to do a surfer/squid version of `plecos` or change this method to adjust the metadata before being sent to `plecos` for validation.